### PR TITLE
Change Microsoft.DiaSymReader portable lib directory to 'portable-net45+win8'

### PR DESCRIPTION
**Pull request by tmat**
@jasonmalinowski

### CODE REVIEW

File: src/Dependencies/Microsoft.DiaSymReader/Version.targets [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <RoslynSemanticVersion>1.0.5</RoslynSemanticVersion>
-    <NuGetVersion>$(RoslynSemanticVersion)</NuGetVersion>
+    <NuGetVersion>$(RoslynSemanticVersion).1</NuGetVersion>
```
Comment (jasonmalinowski) [inline src/Dependencies/Microsoft.DiaSymReader/Version.targets]: How will we know to undo this later?

**Comment by jasonmalinowski** [comment]
:+1:

File: src/Dependencies/Microsoft.DiaSymReader/Version.targets (hunk 1 above)
Comment (tmat) [inline src/Dependencies/Microsoft.DiaSymReader/Version.targets]: We don't need to undo. This component versions separately from Roslyn. Whenever a new version is needed this file is updated. BTW, I'll need to rev the dll at some point to add portable PDB interface.

Comment (jasonmalinowski) [inline src/Dependencies/Microsoft.DiaSymReader/Version.targets]: Oh, then not just update the property above?

Comment (tmat) [inline src/Dependencies/Microsoft.DiaSymReader/Version.targets]: That property above is an assembly version.

Comment (jasonmalinowski) [inline src/Dependencies/Microsoft.DiaSymReader/Version.targets]: ...that's really not obvious. Especially when it's called "RoslynSemanticVersion".

**Pull request merged**